--- Mail-SPF-3.20260331.orig/lib/Mail/SPF/Util.pm	2026-06-12 17:22:57.675513440 +0200
+++ Mail-SPF-3.20260331/lib/Mail/SPF/Util.pm	2026-06-12 17:22:57.666354282 +0200
@@ -161,7 +161,7 @@
 Returns the given I<NetAddr::IP> IPv4 or IPv6 address compactly formatted as a
 I<string>.  For IPv4 addresses, this is equivalent to calling L<NetAddr::IP's
 C<addr>|NetAddr::IP/addr> method.  For IPv6 addresses, this is equivalent to
-calling L<NetAddr::IP's C<short>|NedAddr::IP/short> method.  Throws a
+calling L<NetAddr::IP's C<short>|NetAddr::IP/short> method.  Throws a
 I<Mail::SPF::EInvalidOptionValue> exception if the specified object is not a
 I<NetAddr::IP> IPv4 or IPv6 address object.
 
